packages/api/src/routes/medical/patient.ts

```diff
@@ -310,6 +314,46 @@ router.post(
   })
 );
 
+/**
+ * GET /patient/:id/medical-record
+ *
+ * Returns the url to download a patient's medical record summary, if it exists.
```

Comment:
I'm assuming the logic/flow will be to download a MR from the Dash:
- check if it exists by calling `GET /patient/:id/medical-record-status`
- if it does, call `GET /patient/:id/medical-record`
- if it doesn't, call `POST /patient/:id/consolidated/query` and poll the status w/ `GET /patient/:id/medical-record-status`

Or are we having one for download existing and another for creating a new one?

Reply:
Yea, that's exactly the flow on the dashboard

Reply:
This route, specifically, however, only returns an existing MR Summary. It doesn't trigger creating a new one.